config/initializers/filter_parameter_logging.rb

```diff
@@ -2,6 +2,10 @@
 
 # Configure sensitive parameters which will be filtered from the log file.
 Rails.application.config.filter_parameters += [
+  # Defaults
+  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn,
```

Comment:
I don't think we use any of these params anywhere. Remove?

Reply:
eh, worth keeping em in for the 'start from baseline' reasons detailed above I think